__tests__/app.test.js

```diff
@@ -57,3 +62,30 @@ describe("GET /api/articles/:article_id", () => {
     );
   });
 });
+
+describe('GET /api/articles', () => {
+    test('Get: 200 should respond with an array of all article objects with expected properties', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles; 
+            expect(articles.length).toBeGreaterThan(0)
+            if (articles.length > 0) {
+                articles.forEach(article => {
+                    expect(article).toHaveProperty('author');
+                    expect(article).toHaveProperty('title');
+                    expect(article).toHaveProperty('article_id');
+                    expect(article).toHaveProperty('topic');
+                    expect(article).toHaveProperty('created_at');
+                    expect(article).toHaveProperty('votes');
+                    expect(article).toHaveProperty('article_img_url');
+                    expect(article).toHaveProperty('comment_count');
+                });
+            } else {
+                throw new Error('No articles found in the response');
+            }
```

Comment:
Sorry Danilo, we don't throw errors in the tests. we need to do a Promise.reject in the model and then test for the error.